--- a/lagun/modules/prepareDOE/evaluateDOE.R
+++ b/lagun/modules/prepareDOE/evaluateDOE.R
@@ -13,13 +13,13 @@ get.listDOE <- function(Xopt, lb, ub, ntestMC, callback) {
   dinit <- discrepancyL2centered(Xinit)
   linit <- dchi2LHS(Xinit,nb.samples,0,1)
   pinit <- maxprocrit(Xinit)
-  Xhalton <- runif.halton(nb.samples,nb.inputs)
+  Xhalton <- halton(nb.samples,nb.inputs)
   mhalton <- mindist(Xhalton)
   dhalton <- discrepancyL2centered(Xhalton)
   lhalton <- dchi2LHS(Xhalton,nb.samples,0,1)
   phalton <- maxprocrit(Xhalton)
   callback(1)
-  Xsobol <- runif.sobol(nb.samples,nb.inputs)
+  Xsobol <- sobol(nb.samples,nb.inputs)
   msobol <- mindist(Xsobol)
   dsobol <-  discrepancyL2centered(Xsobol)
   lsobol <- dchi2LHS(Xsobol,nb.samples,0,1)
@@ -171,4 +171,4 @@ evaluateDOE.server <- function(input, output, session, DOE, settings) {
     req(DOE$Xopt, DOE$Xinfos, input$check, checkDOE$listDOE)
     plot.checkDOE.LHS(checkDOE$listDOE)
   })
